notebooks/MAST/JWST/JWST_EngineeringDatabase_Retreival/JWST_EDB_Retrieval.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# JWST Engineering Data Retrieval"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will show you how to retrieve JWST engineeering data and use it in the context of a python session. The [Engineering Data](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.Engineering+Data+v1.3) chapter of the [JWST Archive Manual](https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=DraftMASTDOCS&title=.JWST+Archive+Manual+v1.2) describes how data from thousands of engineering telemetry points on JWST are stored in the Engineering Database in the form of timeseries. These data may be searched by means of an identifier, or **mnenomic**. "
```

Comment:
The first link is pointing to the MASTDocs DRAFT page, we need this link to point to the version in MASTDocs Space.

Reply:
(Minor aside: are we going to capitlize Python throughout? I believe that is the style guide when referring to it as a language, excepth when calling it from the command line of course).

Reply:
- Links fixed
- Whoops. Let's follow Python style guide (I double checked) and capitalize